packages/shared/lib/auxiliary/deep-link/handlers/wallet/operations/handleDeepLinkSendConfirmationOperation.ts

```diff
@@ -20,6 +20,7 @@ import {
 } from '../../../errors'
 import { getAmountFromSearchParam } from '../../../utils'
 import { getByteLengthOfString } from '@lib/utils/getByteLengthOfString'
+import { isStringTrue } from '@lib/utils/isStringTrue'
```

Comment:
We should make a `string.ts` file in `@core/utils` for this function so we don't add to the technical debt by putting this function there.